### DISCUSSION TITLE
Laplacian Mesh Smoothing

### DISCUSSION THREAD

**Discussion by atale480** [pyg-team/pytorch_geometric#7293] [Q&A] [answered]
Is there any method to apply laplacian smoothing on mesh by using pytorch geometric? Trimesh has method for laplacian smoothing but data that I have contain vertices and edge_indices. To convert mesh to triemsh I need vertices and faces. Edge_index can't be converted to face connectivity information again.

**Comment by rusty1s** [comment, marked as answer]
We currently provide `get_mesh_laplacian`. Would that be helpful for you? Otherwise, I suggest to perform smoothing in Trimesh, and then convert to PyG afterwards.
